recipes/aocl-blas/build.sh

```diff
@@ -0,0 +1,16 @@
+CFLAGS=$(echo "${CFLAGS}" | sed "s/-march=[a-zA-Z0-9]*//g")
+CFLAGS=$(echo "${CFLAGS}" | sed "s/-mtune=[a-zA-Z0-9]*//g")
+CXXFLAGS=$(echo "${CFLAGS}" | sed "s/-march=[a-zA-Z0-9]*//g")
+CXXFLAGS=$(echo "${CFLAGS}" | sed "s/-mtune=[a-zA-Z0-9]*//g")
+FFLAGS=$(echo "${CFLAGS}" | sed "s/-march=[a-zA-Z0-9]*//g")
+FFLAGS=$(echo "${CFLAGS}" | sed "s/-mtune=[a-zA-Z0-9]*//g")
+
+cmake -B build -S . \
+      -DBLIS_CONFIG_FAMILY=amdzen \
+      -DBUILD_STATIC_LIBS=OFF \
+      -DENABLE_THREADING=openmp \
+      -DCMAKE_INSTALL_PREFIX=$PREFIX
```

Comment:
```suggestion
      -DCMAKE_INSTALL_PREFIX=$PREFIX \
      ${CMAKE_ARGS}
```
We also want `CMAKE_ARGS` to be used here.

Reply:
Done!